lib/actions/reassemble.js

```diff
@@ -18,12 +28,13 @@ async function timer(this_) {
 
     // eslint-disable-next-line no-await-in-loop
     await this_.emit('data', messages.newMessageWithBody({
-      groupSize: results.messages.length,
+      groupSize: Object.keys(results.messageIdsSeen).length,
       groupId: results.messages[0].groupId,
       messageData: incomingData,
     }));
     // eslint-disable-next-line no-await-in-loop
     await storage.deleteObjectById(groupList[i]);
+    groupList = [];
```

Comment:
Why is this change needed?

Reply:
We need to empty the grouplist after each run to make sure all Base cases in the test environment(reassemble.spec.js) can run properly.

Reply:
:+1: